### PR TITLE
Dashboard pattern

### LINKED ISSUE #2605: Publish Dashboard Pattern to site
Design work done #1947 

Depends on #2606

### PR DESCRIPTION
**Pull request by feliciackh**
closes #2605

preview: https://saltdesignsystem-git-dashboard-pattern-fed-team.vercel.app/salt/patterns/dashboards/analytical-dashboard

26 Oct: dev ready; pending design to provide new/updated images

### CODE REVIEW

File: site/docs/patterns/dashboards/index.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (origami-z) [inline site/docs/patterns/dashboards/index.mdx:1]: This page opens when clicking the menu, which against what [navigation pattern](https://www.saltdesignsystem.com/salt/patterns/navigation#multi-level-group) suggests

File: site/docs/patterns/dashboards/analytical-dashboard.mdx [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+
+The main body of the dashboard and surrounding panels all align to a layout grid.
+
+If your experience is responsive, use the underlying layout grid to set a minimum width for your surrounding panels and adjust your content area based on your predefined breakpoints.
+
+If your experience has a fixed resolution (i.e., isn’t responsive), users can continue to use a layout grid to design consistent layouts.
+
+In the following example, the panel width aligns to the first two columns of the underlying layout grid.
+
+<ImageSwitcher
```
Comment (origami-z) [inline site/docs/patterns/dashboards/analytical-dashboard.mdx:133]: These 2 images have subtle size difference when toggled?

File: site/docs/patterns/dashboards/analytical-dashboard.mdx [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+
+## Dashboard regions
+
+From a high-level overview, the main content regions of an analytical dashboard are:
+
+- Main body region
+- Header region
+- Fixed panels
+
+<ImageSwitcher
```
Comment (origami-z) [inline site/docs/patterns/dashboards/analytical-dashboard.mdx:155]: "Fix panel" words jumped between 2 versions. Also, i don't think using a switch toggle "Visual hierarchy breakdown" is accurately describing the two

File: site/docs/patterns/dashboards/analytical-dashboard.mdx [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+#### Context
+
+- Group contextually related content together where possible.
+- Order modules from level of priority—which data is considered primary vs. secondary for the intended user of the dashboard?
+- Keep the layout content within individual cards as consistent as possible across the dashboard. This supports scalability, especially if main components anchor to familiar locations within a card.
+
+#### Styling
+
+- By default, the Salt [Card](../../components/card) is styled using a primary background. Use the `--salt-container-secondary-background` token to achieve visual hierarchy in your design.
+- _(JPM employees only)_ For styling data visualizations such as charts, follow the UITK guidelines on [Data Visualization](https://uitk.gaiacloud.jpmchase.net/design/data-visualization), where you can find further information on presenting data in a visually cohesive manner. We recommend using the [categorical color palette](https://uitk.gaiacloud.jpmchase.net/design/data-visualization/color#color-palettes:categorical-palette) within your dashboard design, as it’s ideal for visualizing distinct groups of items.
```
[the remark this answers is not in the document]
Comment (feliciackh) [reply site/docs/patterns/dashboards/analytical-dashboard.mdx:238]: have removed both internal link references

File: site/docs/patterns/dashboards/analytical-dashboard.mdx (hunk 3 above)
Comment (feliciackh) [reply site/docs/patterns/dashboards/analytical-dashboard.mdx:155]: I've suggested using "Main body region breakdown" cc @bhoppers2008

File: site/docs/patterns/dashboards/index.mdx (hunk 1 above)
Comment (feliciackh) [reply site/docs/patterns/dashboards/index.mdx:1]: Checked with @lilyvc, Mosaic's side bar is built using third party sidebar and there's this limitations where parent item will navigate as well as expand the group. This behaviour will hopefully be gone when we switch to using Salt's accordion component

**Review by origami-z — APPROVED** [review]

**Review by lilyvc — APPROVED** [review]

**Pull request merged**